ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -475,6 +532,11 @@ bb_exit (void **pdata)
 {
   bounding_boxes *bdata = *pdata;
 
+  if (bdata->is_track) {
```

Comment:
'bdata->is_track' : unnecessary condition (line 940 creates new array instance whether is_track is true or not.)

Reply:
recommend adding release function (g_array_free()) without any condition.

Reply:
I make those array is initialized and freed in decoder init/free function. PTAL